Documentation/DSCResources/VMHostSyslog/VMHostSyslog.md

```diff
@@ -29,7 +28,7 @@ The resource is used to configure the syslog settings on an ESXi node.
 
 Updates the syslog settings of the passed ESXi host.
 
-````powershell
+````Powershell
```

Comment:
Based on this [Document](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/contributing/4-MARKDOWN-SPECIFICS.md) the lower version is valid.